ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -103,7 +103,8 @@ public void ExportToFile(string resourceName, string fileName, VWorld world, Can
                 {
                     continue;
                 }
-                var worldResource = FileLoader.LoadFile(worldNodeName + ".vwnod_c");
+
+                var worldResource = FileLoader.LoadFile(worldNodeName.Replace('\\', '/') + ".vwnod_c");
```

Comment:
This is a bit silly, file loader should probably handle this instead.

Reply:
I'm guessing the tracking one. Sure but it's only this case where the path is malformed.

Reply:
That we know of.